js/main.js

```diff
@@ -276,40 +277,48 @@ $(function() {
     }
   }
 
-  //Create the leaderboard
-  function createLeaderboard(name, score) {
-    var addRow;
-    var winners = [];
-
-    for (var i = 0; i < itemGet.length; i++) {
-      winners[i] = itemGet;
-      console.log(winners[i]);
-    }
-  }
-
   //When player one answer buttons are clicked, it will look for
   function buttonClickPlayerOne(){
-    if($(".answerBtns").hasClass("correct")){
-      $(".correct").css("background-color", "green");
+    ;
+    if($(this).hasClass("correct")){
+      $(".correct").css("background-color", "#00FF0088");
       p1Score++;
       $("#score").html("Score: " + p1Score);
-    } else {
-      $(".answerBtns").css("background-color", "red");
+    }
+    else if ($(this).hasClass("incorrect")) {
+      $(this).css("background-color", "#FF000088");
     }
     $(".answerBtns").off("click");
   }
+
   //When player two answer buttons are clicked, it will look for
   function buttonClickPlayerTwo(){
-    if($(".answerBtns2").hasClass("correct")){
-      $(".correct").css("background-color", "green");
-      p1Score++;
-      $("#score").html("Score: " + p1Score);
-    } else {
-      $(".answerBtns2").css("background-color", "red");
+    if($(this).hasClass("correct")){
+      $(".correct").css("background-color", "#00FF0088");
+      $(".incorrect").css("background-color", "#FF000088");
+      p2Score++;
+      $("#score2").html("Score: " + p2Score);
+    }
+    else if ($(this).hasClass("incorrect")) {
+      $(this).css("background-color", "#FF000088");
     }
     $(".answerBtns2").off("click");
   }
 
+  //Create the leaderboard
+  function createLeaderboard() {
+    var container = $("#leaderboard");
+    var keys = Object.getOwnPropertyNames(localStorage);
+    var values = [];
+
+    for (var items in localStorage){
+      values.push(localStorage[items]);
+    }
+      for (var i = 0; i < keys.length; i++) {
```

Comment:
check indentation

Reply:
Hi Anthony, I made the changes you requested

Thank you